src/Traits/PayPalAPI/PaymentCaptures.php

```diff
@@ -55,4 +55,24 @@ public function refundCapturedPayment(string $capture_id, string $invoice_id, fl
 
         return $this->doPayPalRequest();
     }
+
+    /**
+     * Refund full captured payment
+     *
+     * @param string $capture_id
+     *
+     * @throws \Throwable
+     *
+     * @return array|\Psr\Http\Message\StreamInterface|string
+     *
+     * @see https://developer.paypal.com/docs/api/payments/v2/#captures_refund
+     */
+    public function refundFullCapturedPayment(string $capture_id)
```

Comment:
Need to also add the request body as mentioned here:

https://developer.paypal.com/docs/api/payments/v2/#captures_refund

Reply:
Also the tests are missing.